.babelrc

```diff
@@ -0,0 +1,11 @@
+{
+    "presets": [
+        [
+            "@babel/preset-env"
+        ]
+    ],
+    "plugins": [
+        "@babel/plugin-syntax-dynamic-import",
+        "@babel/plugin-transform-runtime"
+    ]
+}
```

Comment:
![image](https://user-images.githubusercontent.com/14761975/109479554-8d06ad80-7a8b-11eb-9369-995639067572.png)

Reply:
Please, install and use `eslint` which would help you to resolve unfortunate warnings

Reply:
Thanks, done.